server/views/pages/create/checkAnswers.njk

```diff
@@ -28,16 +30,16 @@
                 {% if licence.typeCode === 'AP' %}
                     <h1 class="govuk-heading-l">Vary licence details</h1>
                 {% elseif licence.typeCode === 'AP_PSS' %}
-                    <h1 class="govuk-heading-l">Vary licence and post sentence supervision order details</h1>
+                    <h1 class="govuk-heading-l">Vary post sentence supervision order details</h1>
```

Comment:
Is this correct? This doesn't seem to be checking `inInPssPeriod` so it should be possible for users to vary the AP conditions on an AP_PSS licence

Reply:
reverted this back, as it is not my change